### PR TITLE
RD-1877: extract allowedSettingsTo

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
## Description

[RD-1877](https://cloudifysource.atlassian.net/browse/RD-1877)
- extract `allowedSettingTo` to `Stage.Common.Consts` so it could be share between many widgets instead hardcoded value

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [ ] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2861/
https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2868/
## Documentation

n/a

### CODE REVIEW

File: widgets/sites/src/SitesTable.tsx [hunk 1]
```diff
@@ -234,5 +235,6 @@ SitesTable.propTypes = {
         total: PropTypes.number
     }).isRequired,
     toolbox: Stage.PropTypes.Toolbox.isRequired,
-    widget: Stage.PropTypes.Widget.isRequired
+    widget: Stage.PropTypes.Widget.isRequired,
+    allowedSettingTo: PropTypes.arrayOf(PropTypes.string)
```
Comment (kubama) [inline widgets/sites/src/SitesTable.tsx:239]: Not sure why it was added, I can see no references to `allowedSettingTo` prop

File: widgets/secrets/src/SecretsTable.tsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ function SecretValue({
     secretKey,
     onHide,
     onShow,
-    toolbox
+    toolbox,
+    allowedSettingTo = Stage.Common.Consts.allowedSettingTo
```
Comment (kubama) [inline widgets/secrets/src/SecretsTable.tsx:16]: Not sure why it is added as a prop, I can see no places where this prop is actually set

File: widgets/common/src/deployments/DeploymentDetails.tsx [hunk 3]
```diff
@@ -34,7 +34,8 @@ export default function DeploymentDetails({
     deployment,
     instancesCount,
     instancesStates,
-    onSetVisibility
+    onSetVisibility,
+    allowedSettingTo = Stage.Common.Consts.allowedSettingTo
```
Comment (kubama) [inline widgets/common/src/deployments/DeploymentDetails.tsx:38]: Not sure why it is added as a prop, I can see no places where this prop is actually set

File: widgets/blueprints/src/BlueprintsTable.tsx [hunk 4]
```diff
@@ -10,7 +10,8 @@ export default function BlueprintsTable({
     onSelectBlueprint,
     onSetVisibility,
     toolbox,
-    widget
+    widget,
+    allowedSettingTo = Stage.Common.Consts.allowedSettingTo
```
Comment (kubama) [inline widgets/blueprints/src/BlueprintsTable.tsx:14]: Not sure why it is added as a prop, I can see no places where this prop is actually set

File: widgets/blueprints/src/BlueprintsCatalog.tsx [hunk 5]
```diff
@@ -12,7 +12,8 @@ export default function BlueprintsCatalog({
     onSelectBlueprint,
     onSetVisibility,
     widget,
-    toolbox
+    toolbox,
+    allowedSettingTo = Stage.Common.Consts.allowedSettingTo
```
Comment (kubama) [inline widgets/blueprints/src/BlueprintsCatalog.tsx:16]: Not sure why it is added as a prop, I can see no places where this prop is actually set

File: widgets/common/src/Consts.ts [hunk 6]
```diff
@@ -22,7 +22,8 @@ const consts = {
         } as MapOptions,
         initialZoom: 2.5,
         urlTemplate: '/maps/{z}/{x}/{y}/{r}'
-    }
+    },
+    allowedSettingTo: ['tenant', 'global'] as string[]
```
Comment (kubama) [inline widgets/common/src/Consts.ts:26]: I think the const name `allowedSettingTo` doesn't properly describe the meaning of the value. If I understand correctly it is a list of possible values for visibility setting. If so, I think a better name would be something like `allowedVisibilitySettings`.

File: widgets/secrets/src/SecretsTable.tsx (hunk 2 above)
Comment (PatrycjaKubiczek) [reply widgets/secrets/src/SecretsTable.tsx]: fixed

File: widgets/blueprints/src/BlueprintsCatalog.tsx (hunk 5 above)
Comment (PatrycjaKubiczek) [reply widgets/blueprints/src/BlueprintsCatalog.tsx]: changed the way I added value from props to imports, thank you for noticing it 👍

File: widgets/common/src/Consts.ts (hunk 6 above)
Comment (PatrycjaKubiczek) [reply widgets/common/src/Consts.ts]: great suggestion, changed, thanks!:)

**Review by NorbertAmartus — DISMISSED** [review]
Looks good to me :)
Nicely done 🎉

**Review by kubama — APPROVED** [review]

**Comment by PatrycjaKubiczek** [comment]
@NorbertAmartus @kubama I needed to make some adjustments due to discovered bugs while running tests and because of many failed tests on Jenkins it took me some time to figure it out, but I think everything is okay right now and I can proceed with it.

https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2900/ the same test are failing on master branch

**Review by kubama — APPROVED** [review]

**Pull request merged**